README.md

```
@@ -1,2 +1,1026 @@
+license = {text = "MIT"}
+
+
+[tool.pdm]
+distribution = false
+```
+
+A simple example of a `run.yaml` file can be found [here](examples/run.yaml)
+
+To run Llama Stack perform these two commands:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)